cg/models/flow_cell/flow_cell.py

```diff
@@ -238,6 +268,42 @@ def is_flow_cell_ready(self) -> bool:
         LOG.info(f"Flow cell {self.id} is ready for downstream processing")
         return True
 
+    def get_run_completion_status(self) -> Path | None:
+        """Return the run completion status path."""
```

Comment:
I am not a massive fan of this potentially returning None. Feel free to disregard, but I think it might be cleaner to either handle the .exists() in the caller or to raise an error if the behaviour is unexpected.

Reply:
only novaseqx flow cells have this file. so it should just return none in these cases.